### ISSUE TITLE
 connecting to bluetooth under OS X

### ISSUE THREAD

**Issue by mpble** [mooltipass/minible#42]
Use one of the two templates below and delete the other. All issues not following this template will be directly closed.

8<------------------------ BUG REPORT -----------------------------------------
Under Catalina (OS X) bluetooth pairing with a known device.
Activate bluetooth under OS X, Catalina; activate bluetooth on Mooltipass BLE.
### Expected behavior
See a stable connection between Mooltipass and MacBook Pro.
### Actual behavior
Connection is instable. Only after removing the device in OS X and adding it again the connection is stable.
### Step by step guide to reproduce the problem
- activate bluetooth under OS X
- activate bluetooth on Mooltipass
- remove the device under OS X
- add the device under OS X again

### Moolticute Version - If Involved
0.42.9
### Operating System
MacBook Pro, Model 2017, MacOS Catalina latest stable (10.15.1)
8<------------------------ END BUG REPORT -------------------------------------

**Comment by limpkin** [comment]
closing the issue as the code involved in this one isn't there anymore.
Once you receive our upcoming firmware update, feel free to open a new issue if it still exists
